charts/etcd/templates/etcd-compaction-cronjob.yaml

```diff
@@ -45,12 +45,12 @@ spec:
           restartPolicy: Never
           containers:
           - name: compact-backup
-            image: {{ .Values.backup.image }} 
+            image: {{ .Values.backup.image }}
             imagePullPolicy: {{ .Values.backup.pullPolicy }} 
             command:
             - etcdbrctl
             - compact
-            - --data-dir=/var/etcd/data/new.etcd
+            - --data-dir=/var/etcd
```

Comment:
This path would depend on the volume mount, right? Can you please add the volume mount (`emptyDir`) for the data as well as the snapshot temp files (as discussed in https://github.com/gardener/etcd-backup-restore/pull/367#issuecomment-894004133).

Reply:
I was thinking in the lines of a separate PR adding the volumemount `emptyDir` (maybe with or without tempfs), `--snapstore-temp-directory`, as well as custom resources specific to the compaction cronjob that I would raise after feedback from perf test colleagues.

This was to keep issues separated
However, I could add those changes in this PR if that is preferable. WDYT?

Reply:
I am fine either way 👍